docs/release-notes.rst

```diff
@@ -6,6 +6,98 @@
+   resources.
+
+-  WebUI: Add the ability for administrators to use the CLI to set a message to be displayed on all
+   pages of the WebUI (for example, ``det master cluster-message set -m "Your message"``). Optional
+   flags are available for scheduling the message with a start time and an end time. Administrators
+   can clear the message anytime using ``det master cluster-message clear``. Only one message can be
+   active at a time, so setting a new message will replace the previous one.
+
+-  Kubernetes: Add a feature where Determined offers the users to provide custom Checkpoint GC pod spec.
+      This configuration is done using the ``task_container_defaults.checkpointGcPodSpec`` field
+      within your ``value.yaml`` file. User can create a custom pod specification for CheckpointGC,
+      it will override the default experiment's pod spec settings. Determined by default uses the
+      experiment's pod spec, but by providing custom pod spec users have the flexibility to
+      customize and configure the pod spec directly in this field. User can tailor the garbage
+      collection settings according to the specific GC needs.
+
+-  Kubernetes: The :ref:`Internal Task Gateway <internal-task-gateway>` feature enables Determined
+   tasks running on remote Kubernetes clusters to be exposed to the Determined master and proxies.
+   This feature facilitates multi-resource manager setups by configuring a Gateway controller in the
+   external Kubernetes cluster.
+
+.. important::
+
+   Enabling this feature exposes Determined tasks to the outside world. It is crucial to implement
+   appropriate security measures to restrict access to exposed tasks and secure communication
+   between the external cluster and the main cluster. Recommended measures include:
+
+      -  Setting up a firewall
+      -  Using a VPN
+      -  Implementing IP whitelisting
+      -  Configuring Kubernetes Network Policies
+      -  Employing other security measures as needed
+
+-  Kubernetes Configuration: Allow Cluster administrators to define Determined resource pools on
+   Kubernetes using node selectors and/or affinities. Configure these settings at the default pod
+   spec level under ``task_container_defaults.cpu_pod_spec`` or
+   ``task_container_defaults.gpu_pod_spec``. This allows a single cluster to be divided into
+   multiple resource pools using node labels.
+
+-  WebUI: Allow resource pool slot counts to reflect the state of the entire cluster. Allow slot
+   counts and scheduling to respect node selectors and affinities. This impacts Determined clusters
+   deployed on Kubernetes with multiple resource pools defined in terms of node selectors and/or
+   affinities.
+
+**Bug Fixes**
+
+-  Kubernetes: Fix an issue where where jobs would remain in "QUEUED" state until all pods were
+   running. Jobs will now correctly show as "SCHEDULED" once all pods have been assigned to nodes.
+
+**ZZZ put these into an appropriate section!**
+
```

Comment:
:-)

Reply:
its ok to leave `ZZZ`?

Reply:
we should remove it. i thought it was a reminder for someone? maybe not.

Reply:
fixed